handbook.md

```diff
@@ -236,6 +236,61 @@ As a maintainer, your responsibility is to keep a project running, taking care o
+
+*  Support users and other developers on the mailing lists, in Redmine, Serverfault, and in IRC #theforeman and #theforeman-dev
+*  Review and test the patches submitted by others; this helps to offload the burden on existing committers, who will definitely appreciate your efforts
+*  Participate in discussions about releases, roadmaps, architecture, and long-term plans
+*  Improve the website
+*  If possible, participate in (or even initiate) real-world events such as user/developer meetups, papers/talks at conferences, etc
+*  Improve project infrastructure in order to increase the efficiency of committers and other contributors
+*  Help raise the project's quality bar (e.g. by setting up code coverage analysis)
+*  As much as possible, keep your activity sustained rather than sporadic
+
+#### How do I become a committer?
+
+One person has to nominate you to the group of existing committers.
+The person who nominates you has to:
+
+  * Submit 10 examples that prove this person behaves like a committer. For instance:
+    * Strategic patches to the project that fix or add key functionality
+    * Mailing list threads where the person is clearly helpful
+    * Comments that proof the person has triaged bugs
+    * Proof the nominee has maintained a plugin successfully
+    * Anything else that shows the nominee should become a committer
+  * Explain how the nominee is involved in the community and cares about the future of the project
+
+After the nomination is submitted, two other committers have to second your nomination. If no one objects in one week, you're a committer.
+If anyone objects or wants more information, that person needs to argue for the objection. After that, the committers discuss and come to a consensus within the next week.
+
+In the event of a failed nomination, feedback is reported back to the nominee to help them improve. Feedback comes from a spokesperson from the whole committers group, not just an individual, to avoid potential rancor between the rejected nominee and an existing committer.
+
+If you are inactive in the community for one year, we will remove you from the committers list and revoke your permission, but we will make a mention of you in theforeman.org list of previous committers.
+
+In the event that a committer continues to disregard good citizenship (or actively disrupts the project), we may need to revoke that person's status. The process is the same as for nominating a new committer: someone suggests the revocation with a good reason, two people second the motion, and a vote may be called if consensus cannot be reached. We hope that's simple enough, and that we never have to test it in practice.
+
+Special thanks to Chromium, Mozilla, V8, FreeBSD and Apache Hive for providing the basis of this procedure.
+
+#### Quick tips for new committers
+
+ * If something you merged broke something, the build, etc.. you take responsibility
```

Comment:
also, only merge things that pass the tests, even if the tests failure is not related to the current PR  (external cause)